interfaces/HO-Portal/src/app/program/program-people/program-people.component.html

```diff
@@ -1,30 +1,38 @@
 <section *ngIf="componentVisible">
   <h2>{{ 'page.program.program-details.section-people'|translate }}</h2>
   
-  <p *ngIf="!enrolledPeople?.length && !showSensitiveData">
+  <p>
+    {{ 'page.program.program-people.targetNumber'|translate }} {{program?.highestScoresX}}
+  </p>
```

Comment:
Not all programs will have the property `highestScoreX` (only programs of that 'flavour').
So the template should support all possible versions of a `program`, so "only show this if this feature/property is available".

```suggestion
  <p *ngIf="program?.highestScoresX">
    {{ 'page.program.program-people.targetNumber'|translate }} {{program?.highestScoresX}}
  </p>
```